pep-9999.rst

```diff
@@ -0,0 +1,506 @@
+PEP: 9999
+Title: Dependency specification in pyproject.toml using an exploded TOML table
+Author: Laurie Opperman <laurie_opperman@hotmail.com>,
+        Arun Babu Neelicattu <arun.neelicattu@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 02-09-2020
+Post-History:
+
+
+Abstract
+========
+
+This PEP specifies how to write a project's dependencies in a
+``pyproject.toml`` file for packaging-related tools to consume using the fields
+defined in :pep:`621`, as an alternative to the :pep:`508`-based approach
+defined in :pep:`631`. Specifically, the tables ``project.dependencies`` and
+``project.optional-dependencies`` are defined.
```

Comment:
```suggestion
defined in :pep:`631`.
```

I don't think that sentence is necessary and it's a little hard to follow what you mean without understanding the full context of PEP 631.